lxd/images.go

```diff
@@ -230,14 +265,15 @@ func getImageMetadata(fname string) (*imageMetadata, error) {
 	output, err := exec.Command("tar", args...).Output()
 
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("Could not get image metadata: %v", err)
 	}
 
 	metadata := new(imageMetadata)
-	err = json.Unmarshal(output, &metadata)
+	err = yaml.Unmarshal(output, &metadata)
 
 	if err != nil {
-		return nil, err
+		fmt.Println(err)
```

Comment:
Can we axe the Println here? I know we've got a bunch of these in this code, but we should focus on removing them.